common/src/reserved.rs

```diff
@@ -66,7 +72,125 @@ impl ReservedState {
     }
 
     pub fn apply_delegate(&mut self, _tx: &TxDelegate) -> Result<Self, String> {
```

Comment:
```suggestion
    pub fn apply_delegate(&mut self, tx: &TxDelegate) -> Result<Self, String> {
```
Fix on all occurrence please.